### PR TITLE
fix: copy the description of the option to its alias in completion

### PR DESCRIPTION
**Pull request by mshrtsr**
This PR fixes https://github.com/yargs/yargs/issues/2268.  
I find the original option for the alias and copy its description to the alias.
As a result, the option and its alias are collected together and aligned in the completion.

### CODE REVIEW

File: lib/completion.ts [hunk 1]
```diff
@@ -258,7 +258,9 @@ export class Completion implements CompletionInstance {
     if (!this.zshShell) {
       completions.push(dashes + key);
     } else {
-      const desc = descs[key] || '';
+      const aliasKey = this?.aliases?.[key]?.[0];
```
Comment (bcoe) [inline lib/completion.ts:261]: what about cases where there are multiple aliases associated with a key? should this be a loop?

**Review by bcoe — CHANGES_REQUESTED** [review]

File: lib/completion.ts (hunk 1 above)
Comment (mshrtsr) [reply lib/completion.ts:261]: Oh, Thank you. Yes, sometimes the `this?.aliases?.[key]?.[0]` will be another alias without a description.
I will make it a loop.

**Comment by mshrtsr** [comment]
@bcoe 
Thank you.
Can I ask you for the review again?

**Review by bcoe — APPROVED** [review]

**Comment by bcoe** [comment]
@mshrtsr thank you for the contribution :ok_hand:

**Pull request merged**